dask_jobqueue/tests/test_slurm.py

```diff
@@ -8,7 +8,10 @@
 
 
 def test_header():
-    with SLURMCluster(walltime='00:02:00', processes=4, threads=2, memory='7GB') as cluster:
+    with SLURMCluster(walltime='00:02:00', processes=4, threads=2,
+                      memory='7GB') as cluster:
+
+        print(cluster.job_header)
```

Comment:
Debug print statements?

Reply:
yes, I'm using CI to debug a few things at the moment. I'll ping folks here when I have things cleaned up.